.storybook/main.js

```diff
@@ -17,18 +23,14 @@ module.exports = {
   ],
   framework: "@storybook/react",
   webpackFinal: async (config) => {
-    return {
-      ...config,
-      resolve: {
-        ...config.resolve,
-        alias: {
-          // (thuang): Temp fix for Github issue:
-          // https://github.com/storybookjs/storybook/issues/10231#issuecomment-728038867
-          ...config.resolve.alias,
-          "@emotion/core": toPath("node_modules/@emotion/react"),
-          "emotion-theming": toPath("node_modules/@emotion/react"),
-        },
-      },
+    // (thuang): Temp fix for Github issue:
+    // https://github.com/storybookjs/storybook/issues/10231#issuecomment-728038867
+    config.resolve.alias = {
```

Comment:
I'd prefer creating a new object to return than modifying args, but not sure if y'all do it differently

Reply:
Agreed, I'll just roll it back :D